.config/cargo_spellcheck.toml

```diff
@@ -0,0 +1,11 @@
+[Hunspell]
+lang = "en_US"
+search_dirs = ["."]
+extra_dictionaries = ["cargo_spellcheck.dic"]
+
+[Hunspell.quirks]
+transform_regex = [
+	"^\\s+- $", # for lists in doc comments we ignore the leading '-'
```

Comment:
@drahnr Do you have a better idea of fixing this?

Reply:
Can you given an example where this is necessary?

```
Zyxels zyxeled
- alpha
- beta
List zyxelation lists:

- gamma
- delta
```

does not yield errors I'd not expect.

Reply:
Ah turns out it's no longer necessary if I supply `--checkers hunspell` on the cli.